_maps/RandomRuins/SpaceRuins/sos4.dmm

```diff
@@ -26,10 +26,6 @@
 /obj/structure/chair/comfy/shuttle{
 	dir = 8
 	},
-/obj/effect/mob_spawn/human/corpse/assistant{
```

Comment:
Replace with /obj/effect/decal/remains/human or /obj/effect/decal/cleanable/blood/gibs/up?

Reply:
?wyci
im sure people have gotten used to finding unscathed assistant corpses in maint
not even related to pr